worldengine/cli/main.py

```diff
@@ -302,6 +306,8 @@ def main():
     g_generate.add_argument('--not-fade-borders', dest='fade_borders', action="store_false",
                                help="Not fade borders",
                                default=True)
+    g_generate.add_argument('--scatter', dest='scatter_plot',
```

Comment:
perhaps it would make more sense to have it as a separate operation, like drawing the ancient map. We could also want to change the name because scatter plot is a type of plot but it does not tell us what kind of information is being represented. Later we could have differe scatter plots representing different things